templates/infos/mentions.html.twig

```diff
@@ -11,11 +11,19 @@
         </div>
         <div class="list-group mt-2">
             <div class="flex-column align-items-start mb-4">
-                <small class="text-muted">Page en construction.</small>
+                <small class="text-muted"></small>
             </div>
             <div class="flex-column align-items-start mb-4">
-                <p class="mb-1">Titre</p>
-                <small class="text-muted">Informations.</small>
+                <p class="mb-1">Responsable de publication</p>
+                <small class="text-muted">Bipbip Mobile est édité par la société Enviro Services France.<br><br>
```

Comment:
please not br

Reply:
can't find other way than br with bootstrap, I will search later.